MediaBrowserTv/app/src/main/res/values-cs/strings.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
I would say move the string value changes to their own commit for best practices, makes automated translation services much easier to work with. Weblate and similar programs can really screw things up after a merge.

Reply:
Not really sure how I would?

Reply:
You are right, it would be hard now that it's not the latest commit on your branch. Not need to separate the strings out for this PR.